docs/Documentation/Applications/ansys.md

```diff
@@ -55,29 +31,28 @@ To launch Ansys Fluent jobs in parallel batch mode, you can build on the batch s
     #!/bin/bash
     ...
     #SBATCH --nodes=2
-    #SBATCH --ntasks-per-node=36
+    #SBATCH --ntasks-per-node=104
 
     cd $SLURM_SUBMIT_DIR
-    module purge  # purge everything else
-    module load ansys/<version>
-    module load intel-mpi/2018.0.3
+    module load PrgEnv-intel
+    module load ansys/2023R1
+    
     ...
-    unset I_MPI_PMI_LIBRARY
-    srun hostname -s | sort -V > myhosts.txt
+    NODEFILE=${SLURM_SUBMIT_DIR}/slurmhosts.txt
     ...
-    fluent 2ddp -g -t $SLURM_NTASKS -cnf=myhosts.txt -mpi=intel -pinfiniband -i input_file.jou
+    fluent 2ddp -pcheck -mpi=intel -t$SLURM_NTASKS -cnf=$NODEFILE -gu -driver null -i input_file.jou > output_file.jou
     ```
 
-Once this script file (assumed to be named `ansys-job.slurm`) is saved, it can be submitted to the job scheduler with
+Once this script file (assumed to be named `submit_ansys_job.sh`) is saved, it can be submitted to the job scheduler with
 
 ```
-[user@el3 ~]$ sbatch ansys-job.slurm
+[user@kl3 ~]$ sbatch submit_ansys_job.sh
 ```
 
-In this example batch script, `2ddp` can be replaced with the version of FLUENT your job requires (`2d`, `3d`, `2ddp`, or `3ddp`), `-g` specifies that the job should run without the GUI, `-t` specifies the number of processors to use (in this example, 2 x 36 processors), `-cnf` specifies the hosts file (the list of nodes allocated to this job), `-mpi` and `-p<...>` specify the MPI implementation and interconnect, respectively, and`-i` is used to specify the job input file.  Note that generally speaking the generation of the hostname file,`myhosts.txt`, must be repeated in the beginning of each job since the allocated nodes will likely change for each run. 
+In this example batch script, `2ddp` can be replaced with the version of FLUENT your job requires (`2d`, `3d`, `2ddp`, or `3ddp`), `-g` specifies that the job should run without the GUI, `-t` specifies the number of processors to use (in this example, 2 x 36 processors), `-cnf` specifies the hosts file (the list of nodes allocated to this job), `-mpi` and `-p<...>` specify the MPI implementation and interconnect, respectively, and`-i` is used to specify the job input file.  Note that generally speaking the generation of the hostname file,`slurmhosts.txt`, must be repeated in the beginning of each job since the allocated nodes will likely change for each run. 
```

Comment:
```suggestion
In this example batch script, `2ddp` can be replaced with the version of FLUENT your job requires (`2d`, `3d`, `2ddp`, or `3ddp`), `-g` specifies that the job should run without the GUI, `-t` specifies the number of processors to use (in this example, 2 x 104 processors), `-cnf` specifies the hosts file (the list of nodes allocated to this job), `-mpi` and `-p<...>` specify the MPI implementation and interconnect, respectively, and`-i` is used to specify the job input file.  Note that generally speaking the generation of the hostname file,`slurmhosts.txt`, must be repeated in the beginning of each job since the allocated nodes will likely change for each run. 
```